http/stylesheets/_nav.scss

```diff
@@ -1,5 +1,30 @@
 @use 'common/colors';
 
+// Common Nav properties across the site
+nav {
+  .nav-item {
+    margin: 0 1px;
+    text-align: center;
+    box-shadow: none;
+    transition: box-shadow linear 100ms;
+
+    // Fix to prevent a 1px box shadow from showing up on the left or right sized when resizing the page
+    clip-path: inset(0px 1px 0px 1px);
+  }
+
+  .nav-item:first-of-type {
+    margin-left: -1px;
+  }
+
+  .nav-item:hover {
+    box-shadow: inset 0 -0.25rem colors.$nav_hover-highlight-color;
+  }
+
+  .nav-item-active {
+    box-shadow: inset 0 -0.25rem colors.$nav_active-color !important;
```

Comment:
i feel like `!important` is usually a smell. why is it needed here?

Reply:
You're right it usually is code smell and thinking about it more, this can probably be modified to be removed.

In this case `class1:hover` is more specific than `.class2` so `.class1:hover` takes precedence. The `!important` was to override that, since I didn't want to make `.nav-item-active` more specific.
But with our current use case, I can change that to `.nav-item.nav-item-active` to make it work without `!important` so I'll just make that change